deeds-dapp-service/src/main/java/io/meeds/deeds/web/filter/RequestDispatcherFilter.java

```diff
@@ -71,6 +80,17 @@ public class RequestDispatcherFilter extends HttpFilter {
                                                                                             "/farm",
                                                                                             "/tokenomics");
 
+  protected static final List<String>        DAPP_PATHS_FR                  = Arrays.asList("/place-de-marche",
+                                                                                            "/tenants",
```

Comment:
Path has to be different from EN paths to be able to detect language and propose a real alternate URL for FR language.

Reply:
done!